evm-contracts/src/v0.7/dev/OperatorForwarder.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "../interfaces/OperatorInterface.sol";
+
+contract OperatorForwarder {
+  address public immutable authorizedSender1;
+  address public immutable authorizedSender2;
+  address public immutable authorizedSender3;
+
+  address public immutable linkAddr;
+
+  constructor(address link) {
+    linkAddr = link;
+    address[] memory authorizedSenders = OperatorInterface(msg.sender).getAuthorizedSenders();
+    authorizedSender1 = authorizedSenders[0];
```

Comment:
I think in place of `authorizedSenders[0]` we should use `msg.sender.owner()`. Then we can remove the current validation we have where we require that an authorizedSender is always set. This would be simpler on the Operator and the Forwarder.